css/popup.scss

```diff
@@ -550,13 +554,65 @@ body {
     margin: 0;
     padding-bottom: 0;
 
+    .icon__arrow {
+        margin-top: 4px;
+        right: 15px;
+    }
+
     .top-blocked__li {
         z-index: 0;
         position: relative;
         margin-left: 20px;
         margin-right: 20px;
         clear: both;
 
+       &.top-blocked__li--truncated {
+           float: left;
+           clear: none;
+           margin-left: 16px;
+           margin-right: 0;
+           padding-top: 0;
+       }
+
+       &.top-blocked__li--truncated:first-child {
+           margin-left: 23px;
+       }
+
+       .top-blocked__pill {
+           border-radius: 1em;
```

Comment:
We're not using em's at all. So far (for speed of development) we've only used px for everything. When/if we switch to em's, i'd like to use rem vs. em.